pkg/virt-controller/watch/vm.go

```diff
@@ -436,7 +376,7 @@ func (c *VMController) handleDataVolumes(vm *virtv1.VirtualMachine, dataVolumes
 		}
 		if !exists {
 			// Don't create DV if PVC already exists
-			pvc, err := c.getPersistentVolumeClaimFromCache(vm.Namespace, template.Name)
+			pvc, err := typesutil.GetPersistentVolumeClaimFromCache(vm.Namespace, template.Name, c.pvcInformer)
```

Comment:
I do not like the name 	**typesutil** "kubevirt.io/kubevirt/pkg/storage/types".

What do you think about **storage**, or **storagetypes**